code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -689,3 +689,12 @@ var/datum/disease2/disease/wizarditis = null
 /obj/item/weapon/reagent_containers/glass/bottle/carppheromones/New()
 	..()
 	reagents.add_reagent(CARPPHEROMONES, 30)
+
+/obj/item/weapon/reagent_containers/glass/bottle/mindbreaker
+	name = "Mindbreaker Toxin Bottle"
+	desc = "A bottle filled with a strangely low-viscosity fluid. A small label displays its contents something that will cauase a fun time..."
```

Comment:
First of all, typo on the "cause" word. Second, the wording on the label part feels really weird, consider replacing it with: "There's a small decal of a party popper going off." or similar, to imply the !!!FUN!!! and !!!PARTY!!! aspect more indirectly.

Reply:
ok